app/model/editions/EditionsArticle.scala

```diff
@@ -1,9 +1,15 @@
 package model.editions
 
+import com.gu.editions.PublishedArticle
 import play.api.libs.json.Json
 import scalikejdbc.WrappedResultSet
 
-case class EditionsArticle(pageCode: String, addedOn: Long)
+case class EditionsArticle(pageCode: String, addedOn: Long) {
```

Comment:
Why a `Long` and not a `DateTime` here?

Reply:
It’s what the client expects

Reply:
Can we block @itsibitzi until he's back from holiday? The new model uses a `ZonedDateTime` IIRC.